dockstore-client/src/main/java/io/dockstore/client/cli/nested/ApiClientExtended.java

```diff
@@ -236,6 +240,43 @@ public void createBodyPart(MultiPart multiPart, String key, Object formObject) {
                     buildResponseHeaders(response),
                     respBody);
             }
+        } catch (ProcessingException ex) {
+            // This could be caused by a failed Jersey Interceptor/filter, missing message body writers, or other IO exceptions.
+            // Mainly, this error is thrown when the provided WES URL is invalid. For more details, see:
+            // https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/ProcessingException.html
+            errorMessage(MessageFormat.format("There was an error processing the HTTP request: {0}",
+                ex.getMessage()), Client.CONNECTION_ERROR);
+            return null;
+        } catch (ApiException ex) {
+            // Different WES servers provide error messages with different levels of verbosity/usefulness, so include both a default
+            // message and the message provided from the WES server in the printed error.
+            switch (ex.getCode()) {
+            case HttpStatus.SC_BAD_REQUEST:
+                errorMessage(MessageFormat.format("[{0}] The WES request was malformed: {1}",
+                    ex.getCode(), ex.getMessage()), Client.API_ERROR);
+                break;
+            case HttpStatus.SC_UNAUTHORIZED:
+                errorMessage(MessageFormat.format("[{0}] The WES request is unauthorized to be performed on the target WES server: {1}",
```

Comment:
Instead of "the target WES server", you could show the actual server or url. For example, if you had a typo in the url in the ~/.dockstore/config, showing the url could be a helpful debugging tip. Or is this already displayed via `ex.getMessage()`?

Optional or a future ticket, especially given my reservations about this approach.

Reply:
We don't surface the URL. I think that a broader ticket for a WES debug mode would be valuable, especially for dealing with WES-server specific issues. Created a followup ticket: https://github.com/dockstore/dockstore/issues/4709.